config/crd/patches/webhook_in_atlassearchindexconfigs.yaml

```diff
@@ -0,0 +1,17 @@
+# The following patch enables conversion webhook for CRD
+# CRD conversion requires k8s 1.13 or later.
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: atlassearchindexconfigs.atlas.mongodb.com
+spec:
+  conversion:
+    strategy: Webhook
```

Comment:
we have to find out why this is generated. I see these for other types as well. Do you mind to create a ticket?

Reply:
https://jira.mongodb.org/browse/CLOUDP-247307